modules/zmq_application/qb_application/src/qb_application.cpp

```diff
@@ -0,0 +1,71 @@
+#include <iostream>
+#include "queen_bee_svc.h"
+#include <klepsydra/core/yaml_environment.h>
+#include <klepsydra/high_performance/event_loop_middleware_provider.h>
+
+#include <klepsydra/zmq_core/zhelpers.hpp>
+#include <klepsydra/zmq_core/from_zmq_middleware_provider.h>
+#include <klepsydra/zmq_core/to_zmq_middleware_provider.h>
+#include <klepsydra/vision_ocv/image_data_factory.h>
+
+#include <klepsydra/serialization/binary_cereal_mapper.h>
+#include <klepsydra/serialization/json_cereal_mapper.h>
+#include <klepsydra/zmq_vision_ocv/image_event_data_zmq_mapper.h>
+int main(int argc, char **argv)
+{
+
+  std::string yaml_path = "/home/javi/projects/melissa/modules/config/yaml";
+
+  std::string containerName = argc > 1 ? argv[1] : "QBApp";
+  std::string roboBeePort = argc > 2 ? argv[2] : "9001";
+  int adminPort = argc > 3 ? std::atoi(argv[3]) : 9090;
+  int systemPort = argc > 4 ? std::atoi(argv[4]) : 9292;
+  
+  // ZMQ
+  std::string rbUrl = "tcp://*:" + roboBeePort;
+  std::string topic = "image_data";
+
+  spdlog::info("rbURL: {}", rbUrl);
+
+  //  Prepare our context and subscriber
+  zmq::context_t context(1);
```

Comment:
I am missing the creation of the environment. Does not need to be a ```ZMQEnv```, but a normal ```YamlEnv```.